src/test/java/seedu/task/logic/EditCommandTest.java

```diff
@@ -27,11 +27,11 @@ public void execute_editFloatTask_duplicate() throws Exception {
         model.addTask(helper.computingFloatTask());
         model.addTask(helper.computingEditedNameFloatTask());
         Task toBeEdited = helper.computingFloatTask();
-        model.editTask(toBeEdited, 0);
+        model.editTask(toBeEdited, toBeAdded);
```

Comment:
I've actually ignored this test case. It is actually what Xuchen asked above too, as to why I am ignoring it. Can refer to the above comment to see the problem I have with designing this test case. Right now, I am not sure how to change this test case to make it work too. But manually testing indicates that duplicateTaskException is handled appropriately.